README.md

```diff
@@ -71,38 +42,74 @@ options = ExecutionParameters(nshots=1000)
 results = platform.execute([sequence], options)
 
 # Print the acquired shots
-print(results.samples)
+readout_id = sequence.acquisitions[0][1].id
```

Comment:
I think we should explain the meaning of [0][1] and `.id` here.

Reply:
Done in aa113bffe399ed3bb581bcda2e9cbd5b45db59bc.